docs/parameter_tables/PonPyg/TwoSpecies_2L11.csv

```diff
@@ -0,0 +1,7 @@
+Population size,"17,934",Ancestral pop. Size
```

Comment:
Lower case S in Size, just for consistency

Reply:
Good spot - I fixed this when rebasing.